src/test/java/seedu/guest/logic/commands/AddCommandTest.java

```diff
@@ -129,6 +128,11 @@ public boolean hasGuest(Guest guest) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public boolean hasSameRoom(Guest guest) {
```

Comment:
Should we add a duplicate room test as well above?

Reply:
Agreed!